### PR TITLE
Esfoobar usds/835 census tracts geojson

### LINKED ISSUE #835: [SPIKE] Find new geographic boundary data for census tracts
We want to change our data pipeline to use tracts. Currently the census boundary data we use is only block groups, so we should figure out how to get tract boundary data to use instead. Worst case scenario we use geopandas to merge CBG data from existing source, but this would be time intensive and may have issues.

Make sure this includes territory boundaries.

The output of this spike is either completing this work or creating a new issue with more details as a result of the investigation.

### PR DESCRIPTION
**Pull request by esfoobar-usds**
Fixes #835

### CODE REVIEW

File: data/data-pipeline/data_pipeline/etl/sources/census/etl.py [hunk 1]
```diff
@@ -153,22 +156,22 @@ def _load_into_state_csvs(self, fips_code: str) -> None:
             None
         """
         ## write to individual state csv
-        geoid10_list = self.CBG_PER_STATE[fips_code]
+        tractd10_list = self.TRACT_PER_STATE[fips_code]
```
Comment (lucasmbrown-usds) [inline data/data-pipeline/data_pipeline/etl/sources/census/etl.py:159]: Tiny nit: should this be `tractid10_list` instead of `tractd10_list`?

**Review by lucasmbrown-usds — APPROVED** [review]
Overall looks very simple and straightforward! 

Am I right in saying basically this PR just changes the URL and the variable names?

File: data/data-pipeline/data_pipeline/etl/sources/census/etl.py (hunk 1 above)
Comment (esfoobar-usds) [reply data/data-pipeline/data_pipeline/etl/sources/census/etl.py:159]: Absolutely should be!

**Comment by esfoobar-usds** [comment]
> Am I right in saying basically this PR just changes the URL and the variable names?

It does a bit more than that in terms of the data being written, but yes, in summary, we point the Census Tiger data extract to the tract end point and then regenerates all census CSVs to be tract-level.

**Pull request merged**